keps/3122-expose-flavors-in-localqueue-status/kep.yaml

```
@@ -0,0 +1,31 @@
+title: Expose Flavors in LocalQueue Status
+kep-number: 3122
+authors:
+  - "@mbobrovskyi"
+status: implementable
+creation-date: 2024-10-02
+reviewers:
+  - "@mimowo"
+  - "@alculquicondor"
+  - "@tenzen-y"
+approvers:
+  - "@mimowo"
+  - "@alculquicondor"
+  - "@tenzen-y"
+
+# The target maturity stage in the current dev cycle for this KEP.
+stage: beta
+
+# The most recent milestone for which work toward delivery of this KEP has been
+# done. This can be the current (upcoming) milestone, if it is being actively
+# worked on.
+latest-milestone: "v0.9"
+
+# The milestone at which this feature was, or is targeted to be, at each stage.
+milestone:
+  beta: "v0.9"
+
+# List the feature gate name and the components for which it must be enabled
+feature-gates:
+  - name: AvailableFlavors
```

```suggestion
  - name: ExposeFlavorsInLocalQueue
```

Could we use this feature gate based on @mimowo mentioned since the feature gate is global?
This allows us to avoid conflicting with other Flavor specific features.

---

Done